crypto/bindings/field.ts

```diff
@@ -95,12 +95,13 @@ function createFieldBindings(Field: FiniteField) {
     random(): Field {
       return [0, Field.random()];
     },
+    // TODO
```

Comment:
?

Reply:
Can you add an explanation next to the TODO and link with an issue? It will be better to track it then.

Reply:
ok! I think I'll just remove the TODO and make the function throw an error. always better to avoid doing work that has no impact